src/itemrenderer.cpp

```diff
@@ -13,6 +13,7 @@ namespace newsboat {
 bool should_render_as_html(const std::string mime_type)
 {
 	static const std::set<std::string> html_mime_types = {
+		"", // Render as HTML when no mime-type is provided
```

Comment:
That's *very* clever! And also way better than changing the default for the DB column to text/html, because this approach lets us distinguish between "don't know" and "it's definitely HTML" cases. Thank you very much for coming up with this solution!